config/initializers/active_record.rb

```diff
@@ -1,7 +1,8 @@
-if Rails.env.test?
+if Rails.env.test? && !ENV['RUN_SYSTEM_TESTS']
```

Comment:
Ahhh this appears to fix https://github.com/open-path/Green-River/issues/6363... great. I am going to make this change on my branch https://github.com/greenriver/hmis-warehouse/pull/4548.